core/src/main/scala/kafka/log/ProducerStateManager.scala

```diff
@@ -751,6 +751,25 @@ class ProducerStateManager(val topicPartition: TopicPartition,
       None
   }
 
-  private def listSnapshotFiles: Seq[File] = ProducerStateManager.listSnapshotFiles(logDir)
+  private[log] def listSnapshotFiles: Seq[File] = ProducerStateManager.listSnapshotFiles(logDir)
 
+  /**
+   * Remove any producer state snapshot files which do not have a corresponding offset provided
+   * in keepOffsets. The latest snapshot file will always be kept.
```

Comment:
What's keepOffsets?

Reply:
This should be `segmentBaseOffsets`, thanks!